### PR TITLE
Theme change message fixed and added translation support

### PR DESCRIPTION
**Pull request by leofreitas**
**Motivation and Context**
When I changed active theme the message was "Uninstall was successful" and I changed for the default [success0] "Your request was completed successfully." and add translation support to message "The specified theme cannot be found or is active and so cannot be removed."

**How Has This Been Tested?**
Travis and Local

### CODE REVIEW

File: modules/System Admin/theme_manage_uninstallProcess.php [hunk 1]
```diff
@@ -67,9 +67,9 @@
             }
 
             if ($orphaned != 'true') {
-                $URLDelete = $URLDelete.'&return=success0';
-            } else {
                 $URLDelete = $URLDelete.'&return=warning0';
+            } else {
+                $URLDelete = $URLDelete.'&return=success2';
```
Comment (SKuipers) [inline modules/System Admin/theme_manage_uninstallProcess.php]: Has this change flipped the error state? 🤔 It looks like $orphaned being true is a bad thing, so $orphaned != 'true' would be the success state?

**Comment by SKuipers** [comment]
Well caught Leo! I've added one note about `theme_manage_uninstallProcess.php`, otherwise looks great.

**Comment by SKuipers** [comment]
Hi @leofreitas. Any updates on this PR? It looks like the error state in `theme_manage_uninstallProcess.php` has been reversed in these changes. Can you confirm if this is intentional or not? Thanks!

**Comment by leofreitas** [comment]
Hi @SKuipers, I think so. In both situations, deleting the theme in the database is the main action. The difference is that if $ orfaned = true, you do not have to print the file deletion message. And even $ orfaned = true, the action is a delete, and in this case it was successful. What do you think?

**Comment by SKuipers** [comment]
Okay, I think that makes sense. I'm going to merge it and test. Thanks! 👍

**Comment by SKuipers** [comment]
Yep, tested and works great. Thanks for the fix! 😃

**Pull request merged**